src/main/java/se/sundsvall/casedata/service/ConversationService.java

```diff
@@ -0,0 +1,135 @@
+
+		if (!response.getStatusCode().is2xxSuccessful()) {
+			throw Problem.valueOf(INTERNAL_SERVER_ERROR, "Failed to create conversation in Message Exchange");
+		}
+
+		final var messageExchangeId = Optional.ofNullable(response.getHeaders().getLocation())
+			.map(location -> location.getPath().substring(location.getPath().lastIndexOf('/') + 1))
+			.orElseThrow(() -> Problem.valueOf(INTERNAL_SERVER_ERROR, "Failed to create conversation in Message Exchange"));
+
+		// TODO: Create notification
+		final var conversationEntity = toConversationEntity(conversation, municipalityId, namespace, errandId, messageExchangeId);
+		return conversationRepository.save(conversationEntity).getId();
+	}
+
+	public Conversation getConversation(final String municipalityId, final String namespace, final Long errandId, final String conversationId) {
+
+		final var entity = getConversationEntity(municipalityId, namespace, errandId, conversationId);
+
+		final var response = messageExchangeClient.getConversation(municipalityId, messageExchangeNamespace, entity.getMessageExchangeId());
+
+		if (!response.getStatusCode().is2xxSuccessful()) {
+			throw Problem.valueOf(INTERNAL_SERVER_ERROR, "Failed to retrieve conversation from Message Exchange");
+		}
+
+		if (response.getBody() == null) {
+			throw Problem.valueOf(NOT_FOUND, "Conversation not found in Message Exchange");
+		}
+
+		// TODO: Create notification if sequence number is not the latest
+		final var updatedConversation = toConversation(entity, response.getBody());
+		conversationRepository.save(updateConversationEntity(entity, response.getBody()));
+		return updatedConversation;
+	}
+
+	public List<Conversation> getConversations(final String municipalityId, final String namespace, final Long errandId) {
+
+		final var conversations = conversationRepository.findByMunicipalityIdAndNamespaceAndErrandId(municipalityId, namespace, errandId.toString());
+
+		return toConversationList(conversations);
+	}
+
+	public Conversation updateConversation(final String municipalityId, final String namespace, final Long errandId, final String conversationId, final Conversation request) {
+
+		final var entity = getConversationEntity(municipalityId, namespace, errandId, conversationId);
+
+		final var response = messageExchangeClient.updateConversation(municipalityId, messageExchangeNamespace, entity.getMessageExchangeId(), toMessageExchangeConversation(request));
+
+		if (!response.getStatusCode().is2xxSuccessful() || response.getBody() == null) {
+			throw Problem.valueOf(INTERNAL_SERVER_ERROR, "Failed to update conversation in Message Exchange");
+		}
+		// TODO: Create notification
+		conversationRepository.save(entity);
+		return toConversation(entity, response.getBody());
+	}
+
+	public void createMessage(final String municipalityId, final String namespace, final long errandId, final String conversationId, final Message messageRequest, final List<MultipartFile> attachments) {
+		// TODO: Create notification
```

Comment:
Why should each message result in a notification. How will we handle sync of conversation and avoid double notifications?